Dockerfile.ct_server

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ghcr.io/sigstore/scaffolding/ct_server:v1.7.2@sha256:47170e948ac210422de689168a2422c7223f5c45121d910b9168be41deef66d2 AS server
```

Comment:
The tag is incorrect, it should be `v0.7.23` - `crane digest ghcr.io/sigstore/scaffolding/ct_server:v0.7.23` returns `sha256:47170e948ac210422de689168a2422c7223f5c45121d910b9168be41deef66d2`.